chispa/column_comparer.py

```diff
@@ -11,35 +12,33 @@ class ColumnsNotEqualError(Exception):
     pass
 
 
-def assert_column_equality(df, col_name1, col_name2):
-    elements = df.select(col_name1, col_name2).collect()
-    colName1Elements = list(map(lambda x: x[0], elements))
-    colName2Elements = list(map(lambda x: x[1], elements))
+def assert_column_equality(df: DataFrame, col_name1: str, col_name2: str) -> None:
+    rows = df.select(col_name1, col_name2).collect()
+    colName1Elements = list(map(lambda x: x[0], rows))
```

Comment:
If we started to do small changes here, may we rename it to snake case? I would also replace `list(map(lambda))` by list comprehension as a more readable.